src/utilities/colors.scss

```diff
@@ -26,8 +104,8 @@
 .color-border-tertiary  { border-color: var(--color-neutral-muted) !important; }
 .color-border-inverse   { border-color: var(--color-fg-on-emphasis) !important; }
 .color-border-info      { border-color: var(--color-accent-emphasis) !important; }
-.color-border-success   { border-color: var(--color-success-emphasis) !important; }
-.color-border-danger    { border-color: var(--color-danger-emphasis) !important; }
+// .color-border-success   { border-color: var(--color-success-emphasis) !important; }
+// .color-border-danger    { border-color: var(--color-danger-emphasis) !important; }
```

Comment:
These V1 class names didn't change in V2. But to satisfy the "no duplicate class names" linter rule I commented them out.